sequential/utils.py

```diff
@@ -131,7 +131,8 @@ def item_map(items: List[List[str]]) -> (dict, dict):
     :return: a maping from string to ints and the reverse mapping from int to string
     """
     # list[list[events]] -> set[events] Unpack all the events in items into a single list and remove duplicates
-    flat_set = set([item for sublist in items for item in sublist])
+    # Fix the order of items for creating a deterministic item-ID map
+    flat_set = sorted(list(set([item for sublist in items for item in sublist])))
```

Comment:
just checking my understanding: why sort (set) is not enough? And we use sort(list(set)) instead?

Reply:
Yep, you are right. There is no need to use list(). Sort(set) will work and give the same results.